beebop/poppunkWrapper.py

```diff
@@ -89,11 +91,11 @@ def create_microreact(self, cluster: str, internal_cluster: str) -> None:
             distances=self.db_fs.distances,
             rank_fit=None,
             threads=self.args.visualise.threads,
-            output=self.fs.output_microreact(self.p_hash, cluster),
+            output=self.fs.output_visualisations(self.p_hash, cluster),
             gpu_dist=self.args.visualise.gpu_dist,
             deviceid=self.args.visualise.deviceid,
             external_clustering=self.db_fs.external_clustering,
-            microreact=True,
+            microreact=self.args.visualise.microreact,
```

Comment:
Shouldn't this always be true? Don't we always want to generate both microreact and network? (So, the same goes for cytoscape..) Are you leaving it open to allow for projects which only do one or the other?

Reply:
yeah it should but seems like its all done from args.json so I've just placed it as true there.... I'm assuming yeah it opens to more configurability